### ISSUE TITLE
converting makes the cursor go to the end of the line

### ISSUE THREAD

**Issue by la-magbanua** [meganrogge/template-string-converter#28]
Converting a simple string results into the cursor moving to the end which is kinda hassle. This didn't occur in the previous versions.

![image](https://user-images.githubusercontent.com/56512548/99139001-f941da80-266f-11eb-9976-282ab01693f4.png)

**Comment by meganrogge** [comment]
Thanks for letting me know - that’s not supposed to happen.  Can you pls share your settings w me? There have been some recent changes.

**Comment by meganrogge** [comment]
![recording (40)](https://user-images.githubusercontent.com/29464607/99139176-6cedd400-25eb-11eb-8af6-204f760a16d7.gif)


not able to reproduce with the settings that I have.

**Comment by la-magbanua** [comment]
Here are my settings:
{
  "workbench.startupEditor": "newUntitledFile",
  "window.menuBarVisibility": "toggle",
  "editor.fontFamily": "JetBrains Mono",
  "editor.tabSize": 2,
  "editor.wordWrap": "bounded",
  "editor.lineHeight": 24,
  "editor.mouseWheelScrollSensitivity": 1.5,
  "workbench.editor.enablePreview": false,
  "workbench.editor.enablePreviewFromQuickOpen": false,
  "editor.snippetSuggestions": "top",
  "workbench.tree.indent": 10,
  "prettier.arrowParens": "avoid",
  "prettier.semi": false,
  "prettier.singleQuote": true,
  "editor.fontLigatures": true,
  "editor.formatOnSave": true,
  "emmet.includeLanguages": {
    "javascript": "javascriptreact"
  },
  "advancedNewFile.exclude": {
    "node_modules": true,
    "node_modules_electron": true,
    "dev": true,
    "dist": true
  },
  "[jsonc]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
},
"terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe",
"[javascript]": {
  "editor.defaultFormatter": "esbenp.prettier-vscode"
},
}



![vsc](https://user-images.githubusercontent.com/56512548/99139573-ed0c4c00-2674-11eb-8285-552d948bd94e.gif)

**Comment by meganrogge** [comment]
Thanks! Can you pls check what your template string converter extension settings are in the settings UI?

**Comment by la-magbanua** [comment]
I didn't touch any configuration setting of the template string converter. I use it as it is whenever I install it.

**Comment by meganrogge** [comment]
Very strange and since I can’t reproduce, I’d suggest reverting to a prior version of the extension that was working for you using these instructions. https://stackoverflow.com/questions/42626065/vs-code-rollback-extension-install-specific-extension-version/53755378

**Comment by la-magbanua** [comment]
I installed the previous version and it strangely works now. Thanks!

**Comment by Glaived** [comment]
@meganrogge 
I have the same problem

Template String Converter v0.3.0

`code --list-extensions`:

```
adpyke.codesnap
adpyke.vscode-userscript
amatiasq.sort-imports
annsk.alignment
auchenberg.vscode-browser-preview
bar9.stories
bengreenier.vscode-node-readme
bierner.emojisense
BriteSnow.vscode-toggle-quotes
ChakrounAnas.turbo-console-log
christian-kohler.npm-intellisense
chrmarti.regex
CoenraadS.bracket-pair-colorizer-2
cssho.vscode-svgviewer
darxeal.tampermonkey-snippets
dbaeumer.vscode-eslint
dbalas.vscode-html2pug
drKnoxy.eslint-disable-snippets
ecmel.vscode-html-css
EditorConfig.EditorConfig
eg2.vscode-npm-script
enkia.tokyo-night
eriklynd.json-tools
esbenp.prettier-vscode
formulahendry.auto-rename-tag
formulahendry.code-runner
funkyremi.vscode-google-translate
gamunu.vscode-yarn
Gruntfuggly.todo-tree
icrawl.discord-vscode
JakeWilson.vscode-cdnjs
JakeWilson.vscode-placeholder-images
kamikillerto.vscode-colorize
kodetech.electron-debug
mark-wiemer.vscode-autohotkey-plus-plus
masonliu.exercism
meganrogge.template-string-converter
mgmcdermott.vscode-language-babel
mikestead.dotenv
mrmlnc.vscode-duplicate
MS-CEINTL.vscode-language-pack-fr
ms-vscode-remote.remote-wsl
ms-vsliveshare.vsliveshare
msjsdiag.debugger-for-chrome
nickdodd79.gulptasks
octref.vetur
philnash.ngrok-for-vscode
piotrpalarz.vscode-gitignore-generator
PKief.material-icon-theme
richie5um2.vscode-sort-json
ritwickdey.LiveServer
rvest.vs-code-prettier-eslint
ryu1kn.partial-diff
samverschueren.yo
shd101wyy.markdown-preview-enhanced
slevesque.vscode-autohotkey
syler.sass-indented
thekalinga.bootstrap4-vscode
tlevesque.import-to-require
tombonnike.vscode-status-bar-format-toggle
tomoki1207.pdf
VisualStudioExptTeam.vscodeintellicode
vuetifyjs.vuetify-vscode
xyz.local-history
ZainChen.json
zgudino.editorconfig-vscode-snippet
Zignd.html-css-class-completion
```

vscode settings:
```jsonc
{
  "JSON-zain.json.autorefresh": true,
  "[javascript]": {
    "editor.tabSize": 2
  },
  "bracket-pair-colorizer-2.showBracketsInGutter": true,
  "bracket-pair-colorizer-2.unmatchedScopeColor": "#DB4B4B",
  "browser-preview.startUrl": "https://duckduckgo.com",
  "code-runner.clearPreviousOutput": true,
  "code-runner.saveFileBeforeRun": true,
  "codesnap.backgroundColor": "#16161E",
  "codesnap.showWindowControls": true,
  "codesnap.target": "container",
  "codesnap.transparentBackground": true,
  "colorize.colorized_colors": [
    "BROWSERS_COLORS",
    "HEXA",
    "RGB",
    "HSL",
    "ARGB"
  ],
  "colorize.languages": [
    "js",
    "json",
    "jsonc",
    "css",
    "sass",
    "scss",
    "vue",
    "less",
    "postcss",
    "sss",
    "stylus",
    "xml",
    "svg"
  ],
  "diffEditor.codeLens": true,
  "diffEditor.renderSideBySide": false,
  "discord.enabled": false,
  "discord.lowerDetailsEditing": "{workspace}",
  "discord.workspaceElapsedTime": true,
  "editor.cursorBlinking": "phase",
  "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
  "editor.fontFamily": "'Fira Code', Consolas, 'Courier New', monospace",
  "editor.fontLigatures": true,
  "editor.formatOnPaste": true,
  "editor.formatOnSave": true,
  "editor.formatOnType": true,
  "editor.minimap.renderCharacters": false,
  "editor.minimap.size": "fill",
  "editor.renderControlCharacters": true,
  "editor.renderWhitespace": "trailing",
  "editor.smoothScrolling": true,
  "editor.suggestSelection": "first",
  "eslint.codeActionsOnSave.mode": "problems",
  "eslint.packageManager": "yarn",
  "exercism.openStartGlob": "{,!(node_modules)/**/}!(README|babel.config).{dart,fs,js,go,md,rs,vim,cc,cpp,c,coffee,cs,swift,ts,h,hpp,java,lua,rb,sh,vb,php,py}",
  "exercism.openTerminal": false,
  "explorer.confirmDelete": false,
  "files.autoGuessEncoding": true,
  "files.autoSave": "off",
  "files.defaultLanguage": "${activeEditorLanguage}",
  "files.trimFinalNewlines": true,
  "html2pug": {
    "commas": true,
    "doubleQuotes": false,
    "fragment": true,
    "tabs": false
  },
  "liveServer.settings.ChromeDebuggingAttachment": true,
  "liveServer.settings.donotShowInfoMsg": true,
  "liveServer.settings.donotVerifyTags": true,
  "liveServer.settings.host": "localhost",
  "liveServer.settings.useBrowserPreview": true,
  "local-history.dateLocale": "fr-FR",
  "local-history.enabled": 0,
  "markdown-preview-enhanced.codeBlockTheme": "one-dark.css",
  "markdown-preview-enhanced.liveUpdate": true,
  "markdown-preview-enhanced.mermaidTheme": "dark",
  "markdown-preview-enhanced.previewTheme": "none.css",
  "material-icon-theme.activeIconPack": "vue_vuex",
  "material-icon-theme.folders.color": "#bc8cff",
  "material-icon-theme.folders.theme": "specific",
  "material-icon-theme.saturation": 0.9,
  "material-icon-theme.showUpdateMessage": true,
  "npm-intellisense.importLinebreak": "",
  "npm-intellisense.scanDevDependencies": true,
  "npm-intellisense.showBuildInLibs": true,
  "placeholderImages.quoteStyle": "double",
  "powershell.powerShellDefaultVersion": "Windows PowerShell (x86)",
  "prettier.arrowParens": "avoid",
  "prettier.jsxBracketSameLine": true,
  "prettier.jsxSingleQuote": true,
  "prettier.packageManager": "yarn",
  "prettier.semi": false,
  "prettier.singleQuote": true,
  "regex-previewer.enableCodeLens": true,
  "sort-imports.default-sort-style": "eslint",
  "template-string-converter.autoRemoveTemplateString": true,
  "template-string-converter.convertOutermostQuotes": true,
  "terminal.integrated.cursorBlinking": true,
  "terminal.integrated.cursorStyle": "line",
  "terminal.integrated.cursorWidth": 2,
  "terminal.integrated.enableBell": true,
  "terminal.integrated.fontFamily": "'Fira Code'",
  "terminal.integrated.fontSize": 12,
  "terminal.integrated.shell.windows": "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe",
  "todo-tree.filtering.excludeGlobs": [
    "**/node_modules"
  ],
  "todo-tree.filtering.includeHiddenFiles": true,
  "todo-tree.filtering.useBuiltInExcludes": "file excludes",
  "todo-tree.general.enableFileWatcher": true,
  "todo-tree.general.statusBar": "tags",
  "todo-tree.general.tagGroups": {
    "FIX": [
      "FIXME"
    ],
    "TEMP": [
      "TMP"
    ]
  },
  "todo-tree.general.tags": [
    "XXX",
    "TODO",
    "TEMP",
    "HACK",
    "FIX",
    "FIXME",
    "BUG",
    "TMP",
    "DEBUG"
  ],
  "todo-tree.tree.buttons.export": true,
  "todo-tree.tree.expanded": true,
  "todo-tree.tree.showCountsInTree": true,
  "turboConsoleLog.logMessagePrefix": "TCL",
  "turboConsoleLog.quote": "`",
  "update.enableWindowsBackgroundUpdates": false,
  "update.mode": "start",
  "vscodeGoogleTranslate.preferredLanguage": "French",
  "vsintellicode.modify.editor.suggestSelection": "automaticallyOverrodeDefaultValue",
  "workbench.activityBar.visible": true,
  "workbench.colorTheme": "Tokyo Night",
  "workbench.commandPalette.preserveInput": true,
  "workbench.editor.closeOnFileDelete": true,
  "workbench.editor.highlightModifiedTabs": true,
  "workbench.editor.limit.enabled": true,
  "workbench.editor.limit.value": 20,
  "workbench.editor.pinnedTabSizing": "compact",
  "workbench.editor.scrollToSwitchTabs": true,
  "workbench.iconTheme": "material-icon-theme"
}
```

**Comment by meganrogge** [comment]
@Glaived thanks for sharing your settings. Can you see what happens if you set one or both of these to false? 
![CA12FC7F-4235-47CF-A301-5EC561E56706](https://user-images.githubusercontent.com/29464607/99880976-d752d580-2be4-11eb-95cc-599785637c99.jpeg)

**Comment by Glaived** [comment]
| autoRemoveTemplateString | convertOutermostQuotes | Result |
|--------------------------|------------------------|--------|
| ❌                       | ❌                     |😟|
| ❌                       | ✔️                     |😟|
| ✔️                       | ❌                     |😟|
| ✔️                       | ✔️                     |😟|

**Comment by meganrogge** [comment]
So strange that I can’t reproduce the issue. For now, I’d suggest reverting to a prior version that worked and I’ll look into it later today.

**Comment by meganrogge** [comment]
@Glaived @la-magbanua finally able to reproduce and fix should be in very soon!

**Comment by meganrogge** [comment]
Resolved in version 0.3.1, which will be live in a few minutes. Thanks for bringing this to my attention @Glaived @la-magbanua

**Comment by la-magbanua** [comment]
Thanks! It's working perfectly fine now.
